### ISSUE TITLE
Question about watermark decoding performance

### ISSUE THREAD

**Issue by jang-1996** [facebookresearch/stable_signature#26]
Hello, author! Thank you for your wonderful work.

I fine-tuned the LDM decoder using the code and commands you provided. During the training process, the performance on both the Train and Validation sets was nearly 99%, which is very good. However, the Test performance of bit-accuracy was less than 50%, which is not satisfactory.

The testing method is as follows:

1. Replace the original LDM decoder with the fine-tuned decoder : checkpoint_000.pth
2. Generate images using the LDM and extract messages.
3. Compare the extracted messages with the target messages.

Is there something I might have missed or done incorrectly?

Here is my training log.
[log.txt](https://github.com/user-attachments/files/16370639/log.txt)

**Comment by jang-1996** [comment]
This is my testing method 1,2

![스크린샷 2024-07-25 13-39-54](https://github.com/user-attachments/assets/eaba2b36-dd6c-4831-a2f2-52bb31042339)

**Comment by LiRunyi2001** [comment]
Hi there! I am wondering how you solved this issue, thanks a lot!

**Comment by jang-1996** [comment]
Hello!
My setting is 4 GPU. 
In this [issue](https://github.com/facebookresearch/stable_signature/issues/5#issuecomment-1790640271), the author suggests to set new learning rate. As a result, I solve this issue.
I think you should find appropriate learninig rate.
Thank you!

**Comment by fenghe12** [comment]
how did you solved this problem？i encountered similar problems.how did you set lr or other settings?thanks for reply!
